test/unit_tests/braket/experimental/autoqasm/test_api.py

```diff
@@ -294,7 +294,7 @@ def test_bell_measurement_invalid_declared_size() -> None:
     assert expected_error_message in str(exc_info.value)
 
 
-@aq.function
+@aq.function(num_qubits=5)
```

Comment:
It would be pretty cool if `num_qubits` could be accessible inside the function (so that, in this case, `n_qubits` doesn't have to be declared again). But outside the scope of this PR, probably.

Reply:
good idea, should be very easy to implement. could be something like
```
@aq.function(num_qubits=5)
def my_func():
    aq.get_num_qubits()
```